feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio_ubus.c

```diff
@@ -50,6 +50,34 @@ int hapd_rrm_set_neighbors(char *name, struct rrm_neighbor *neigh, int count)
 	return ubus_invoke(ubus, id, "rrm_nr_set", b.head, NULL, NULL, 1000);
 }
 
+int ubus_set_channel_switch(const char *if_name, uint32_t frequency,
+			    int channel_bandwidth, int sec_chan_offset)
+{
+	uint32_t id;
```

Comment:
This function needs to receive the hw_mode parameter so that it can properly select the HT, VHT or HE operation.

Is this handled correctly in the RRM noise/channel occupancy code?